src/index.ts

```diff
@@ -47,7 +47,7 @@ if (typeof _ !== 'undefined' && typeof Dropzone !== 'undefined')
 else HSFileUploadModule = null;
 export { HSFileUploadModule as HSFileUpload };
 
-if (typeof typeof noUiSlider !== 'undefined')
+if (typeof noUiSlider !== 'undefined')
 	HSRangeSliderModule = require('./plugins/range-slider').default;
```

Comment:
Noob question - I am using it for an internal tool and came across this. Why are we using `require` here? Especially when require is not available in browsers and locally linking this package (`pnpm link /local/clone/of/preline`) throws an error that `require` is not defined which it isn't in the browsers if I understand correctly. 

I replaced it with following:

```typescript
	(async () => {
		HSRangeSliderModule = (await import('./plugins/range-slider')).default;
	})();
```

would that be wrong?